examples/README.md

```diff
@@ -0,0 +1,5 @@
+# Examples
+
+Currently there is no examples showing usage of the driver. For examples from the DataStax driver, see ``./DataStax`` directory.
```

Comment:
Is that true? I think the examples from DataStax driver should be valid for our driver too, so there actually **are** examples showing usage of the driver.

Reply:
It will change very soon and what I wanted to say that there is no new examples